include/deal.II/base/data_out_base.h

```diff
@@ -2196,7 +2196,7 @@ namespace DataOutBase
    * piece_names[2].emplace_back("subdomain_01.time_step_2.vtk");
    * piece_names[2].emplace_back("subdomain_02.time_step_2.vtk");
    *
-   * std::ofstream visit_output ("master_file.visit");
+   * std::ofstream visit_output ("parent_file.visit");
```

Comment:
Here and below, let's just call this file `solution.visit`. That's consistent with what we call it in most of the tutorial programs.